streamwebs_frontend/streamwebs/templates/streamwebs/base.html

```diff
@@ -1,19 +1,21 @@
 {% load staticfiles %}
-{% load pipeline %}
-
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="utf-8" />
-    <title>{% block title %}Streamwebs{% endblock %}</title>
+    <title>{% block title %}{% endblock %} StreamWebs</title>
     <link rel="stylesheet" type="text/css" href="{% static 'streamwebs/style.css' %}" />
     <link rel="shortcut icon" href="{% static 'streamwebs/images/favicon.ico' %}" />
-    {% javascript 'libraries' %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
We already have a static version installed via bower. Run `bower install`, then change this to:

    <script src="{% static 'streamwebs/bower_components/jquery/dist/jquery.js' %}"></script>